turbopack/crates/turbopack-core/src/module_graph/chunk_group_info.rs

```diff
@@ -308,17 +304,16 @@ impl Deref for ChunkGroupId {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-struct NodeWithPriority {
+struct TraversalPriority {
     depth: usize,
     chunk_group_len: u64,
```

Comment:
nit:i would expect something called `len` to also be a `usize`... right?

Reply:
I did this because I get that value from `RoaringBitmap::len(&self) -> u64`. I don't know why they chose to do that instead of an `usize` 🤷